common.mk

```diff
@@ -271,6 +275,10 @@ ifeq ($(XEN_BOOT),y)
 DEFCONFIG_XEN=--br-defconfig build/br-ext/configs/xen.conf
 endif
 
+ifneq ($(CFG_MS_TPM_20_REF),y)
```

Comment:
I guess this should be as line 567? I.e., 
```
ifneq ($(CFG_MS_TPM_20_REF),)
```
Since the line (47) setting `CFG_MS_TPM_20_REF` is a path and not `y`.

Reply:
Yes, you're right.